### PR TITLE
Switch to using static images for jetsnack

### PR DESCRIPTION
**Pull request by riggaroo**
Due to unsplash source urls not being available anymore, we've switched to using static Images instead.

### CODE REVIEW

File: Jetsnack/app/src/main/res/drawable-nodpi/almonds.jpg
Comment (JolandaVerhoef) [inline Jetsnack/app/src/main/res/drawable-nodpi/almonds.jpg:1]: Any chance we can optimize these images by lowering their resolution, converting them to webp/png, or running a jpg optimization algorithm on them? 
332KB seems a bit big.

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt [hunk 1]
```diff
@@ -296,7 +298,7 @@ fun SnackImage(
     ) {
         AsyncImage(
             model = ImageRequest.Builder(LocalContext.current)
-                .data(imageUrl)
+                .data(imageRes)
                 .crossfade(true)
                 .build(),
             contentDescription = contentDescription,
```
Comment (JolandaVerhoef) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt:304]: Should you consider removing the placeholder? You removed the jpg so maybe you also want to remove the placeholder parameter here.

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt [hunk 2]
```diff
@@ -296,7 +298,7 @@ fun SnackImage(
     ) {
         AsyncImage(
```
Comment (JolandaVerhoef) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt:299]: Would you still need an asyncImage at all or would you consider using a typical `Image` composable instead?

Comment (riggaroo) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt:299]: AsyncImage loads the image off the main thread, which is better for performance.

File: Jetsnack/app/src/main/res/drawable-nodpi/almonds.jpg
Comment (riggaroo) [reply Jetsnack/app/src/main/res/drawable-nodpi/almonds.jpg:1]: I reduced these quite a bit already, started with large images like 8mb each :D Anyway I reduced a lot of these some more.

**Review by JolandaVerhoef — APPROVED** [review]

**Pull request merged**